.github/workflows/c-cpp.yml

```diff
@@ -10,25 +10,23 @@ jobs:
     strategy:
       matrix:
         config:
-          - {name: i686-pc-windows-msvc, os: windows-latest, cmakeArgs: -G "Visual Studio 16 2019" -A Win32}
           - {name: x86_64-apple-darwin, os: macos-latest, cmakeArgs: -G "Xcode"}
           - {name: x86_64-pc-linux-gnu, os: ubuntu-latest}
-          - {name: x86_64-pc-windows-msvc, os: windows-latest, cmakeArgs: -G "Visual Studio 16 2019" -A x64}
```

Comment:
Why removed tests for VS?

Reply:
Since the MSVC build always fail even on flyinghead's branch, and we are always using MinGW to build.
So I think may be removing it completely would be better since it cannot stop other workflow anymore. 

Anyways may be we can reenable it after we merge with upstream if preferred.